parsers/CA_AB.py

```diff
@@ -116,24 +116,27 @@ def fetch_production(
     return validation.validate(
         {
             "capacity": {
-                "gas": generation["GAS"]["MC"],
-                "hydro": generation["HYDRO"]["MC"],
-                "battery storage": generation["ENERGY STORAGE"]["MC"],
-                "solar": generation["SOLAR"]["MC"],
+                "gas": generation["COGENERATION"]["MC"]
+                + generation["COMBINED CYCLE"]["MC"]
+                + generation["GAS FIRED STEAM"]["MC"]
+                + generation["SIMPLE CYCLE"]["MC"],
                 "wind": generation["WIND"]["MC"],
+                "solar": generation["SOLAR"]["MC"],
+                "hydro": generation["HYDRO"]["MC"],
                 "biomass": generation["OTHER"]["MC"],
-                "unknown": generation["DUAL FUEL"]["MC"],
-                "coal": generation["COAL"]["MC"],
+                "battery storage": generation["ENERGY STORAGE"]["MC"],
             },
             "datetime": get_csd_report_timestamp(response.text),
             "production": {
-                "gas": generation["GAS"]["TNG"],
-                "hydro": generation["HYDRO"]["TNG"],
-                "solar": generation["SOLAR"]["TNG"],
+                "gas": generation["COGENERATION"]["TNG"]
+                + generation["COMBINED CYCLE"]["TNG"]
+                + generation["GAS FIRED STEAM"]["TNG"]
+                + generation["SIMPLE CYCLE"]["TNG"],
```

Comment:
Are we sure all of these are actually gas? Co-Generation could be other things as well from my experience with the other parsers.

Reply:
I also asked myself these questions and I should have put the elements to explain.

We can access to a html version of this data here : 
http://ets.aeso.ca/ets_web/ip/Market/Reports/CSDReportServlet?beginDate=&endDate=&contentType=html


![image](https://github.com/user-attachments/assets/a48408c6-a064-4c20-9675-9477f375e616)
![image](https://github.com/user-attachments/assets/77d57d28-14fb-4357-b95f-4cf405458896)
They group on the left column GAS the 4 sections: 

- Simple Cycle
- Cogeneration
- Combined Cycle
- Gas Fired Steam